incubator/chartmuseum/README.md

```diff
@@ -1,5 +1,78 @@
 # ChartMuseum Helm Chart
 
-Work in progress...
+Deploy your own private ChartMuseum.   
 
-Please see https://github.com/chartmuseum/chartmuseum
\ No newline at end of file
+Please also see https://github.com/chartmuseum/chartmuseum   
+
+## Prerequisites
+
+* Kubernetes with extensions/v1beta1 available
+* [If enabled] A persistent storage resource and RW access to it
+* [If enabled] Kubernetes StorageClass for dynamic provisioning
+
+## Configuration
+
+By default this chart will not have persistent storage.   
+
+For a more robust solution supply helm install with a custom values.yaml   
+You are also required to create the StorageClass resource ahead of time:   
+```
+kubectl create -f /path/to/storage_class.yaml
+```
+
+The following tables lists the configurable parameters of the vault chart and
+their default values.
+
+|       Parameter              |           Description                       |                         Default                     |
+|------------------------------|---------------------------------------------|-----------------------------------------------------|
+| `image.pullPolicy`           | Container pull policy                       | `IfNotPresent`                                      |
+| `image.repository`           | Container image to use                      | `chartmuseum/chartmuseum`                           |
+| `image.tag`                  | Container image tag to deploy               | `0.2.7`                                             |
+| `persistence.accessMode`     | Access mode to use for PVC                  | `ReadWriteOnce`                                     |
+| `persistence.enabled`        | Whether to use a PVC for persistent storage | `false`                                             |
+| `persistence.size`           | Amount of space to claim for PVC            | `8Gi`                                               |
+| `persistence.storageClass`   | Storage Class to use for PVC                | `-`                                                 |
+| `replicaCount`               | k8s replicas                                | `1`                                                 |
+| `resources.limits.cpu`       | Container maximum CPU                       | `100m`                                              |
+| `resources.limits.memory`    | Container maximum memory                    | `128Mi`                                             |
+| `resources.requests.cpu`     | Container requested CPU                     | `80m`                                               |
+| `resources.requests.memory`  | Container requested memory                  | `64Mi`                                              |
+
+Specify each parameter using the `--set key=value[,key=value]` argument to
+`helm install`.
+
+
+## Installation
+
+```shell
+helm install --name my-chartmuseum -f values.yaml incubator/chartmuseum
```

Comment:
This should probably be `-f custom.yaml` or something similar. `values.yaml` should contain defaults then be overridden by another yaml